### ISSUE TITLE
404 error on link for "Connect a New OBO Repository to this form"

### ISSUE THREAD

**Issue by elserj** [Planteome/Ontology-Variable-Request-App#27]
On the bottom of the page is a button for "Connect a New OBO Repository to this form" that is hardcoded in /templates/form.html.j2 to https://github.com/apps/ontology-variable-submission-bot. 

Not sure if the button should be removed or updated.

**Comment by cooperl09** [comment]
I don't think we need this button.  We should be the ones to decide which repos are linked to the form.

**Comment by elserj** [comment]
I've commented it out on the trait-requests.planteome site.

Leaving issue open because it still exists in the code here.
